concepts/src/app.ts

```diff
@@ -1,6 +1,31 @@
 import express from "express";
+import type { Client as ElasticClient } from "@elastic/elasticsearch";
 
-const app = express();
+type Clients = {
+  elastic: ElasticClient;
```

Comment:
Just to check we're on the same page: this structure would allow us to add, say, a `SierraClient` when we do the items and requests APIs?